analyses/connectivity.py

```diff
@@ -180,9 +201,15 @@ def update_matrix(self):
         bordercolor = np.empty(shape, dtype=object)
 
         # call display function on every matrix element
+        total_connected = 0
+        total_probed = 0
+        self.matrix_map = {}
         for i,row in enumerate(cell_groups):
             for j,col in enumerate(cell_groups):
-                output = self.display_func(row, col, results[(row, col)])
+                output = self.display_func(row, col, self.results[(row, col)])
+                self.matrix_map[i, j] = (row, col)
+                total_connected += self.results[(row, col)]['n_connected']
```

Comment:
Something not obvious here: `self.analysis_func` can be reassigned to make the analyzer perform different analyses, so it is not safe to assume that `self.results[(row, col)]` contains the key `n_connected`.